test/e2e/framework.go

```diff
@@ -17,6 +17,7 @@ package e2e
 import (
 	"bytes"
 	"fmt"
+	"github.com/vmware-tanzu/antrea/pkg/agent/types"
```

Comment:
@suwang48404 at the moment tests can be run from macOS (by connecting to an arbitrary K8s cluster running Antrea). I hope this import does not break that?

Reply:
good catch, that would break Mac os test. thx

Reply:
reference to netlink in pkt/agent/types module is removed, so it can safely called on Mac OS